src/Microsoft.ML.Transforms/GroupTransform.cs

```diff
@@ -180,35 +180,42 @@ public override RowCursor[] GetRowCursorSet(Func<int, bool> predicate, int n, Ra
         }
 
         /// <summary>
+        /// The output schema collects columns used to group columns (aka group-key columns) columns being grouped from the source scheam.
```

Comment:
>columns (aka group-key columns) columns [](start = 61, length = 39)

columns columns columns!
I think you missed columns between columns and columns.
Seriously, i think last columns is not necessary.

Reply:
I will rewrite the whole summary.

---
In reply to: [245808197](https://github.com/dotnet/machinelearning/pull/2060#discussion_r245808197) [](ancestors = 245808197)